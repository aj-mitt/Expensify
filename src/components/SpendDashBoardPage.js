import React from 'react'
import SpendingList from './SpendingList'
import SpendingListFilters from './SpendingListFilters'
import SpendingSummary from './SpendingSummary'

const SpendDashboardPage = () => (
  <div>
    <SpendingSummary />
    <SpendingListFilters />
    <SpendingList />
  </div>

)

export default SpendDashboardPage
