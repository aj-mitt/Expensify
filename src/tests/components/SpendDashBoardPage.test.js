import React from 'react'
import {shallow} from 'enzyme'
import SpendDashBoardPage from '../../components/SpendDashBoardPage'

test('Should render spend dashboard', () => {
  const wrapper = shallow(<SpendDashBoardPage />)
  expect(wrapper).toMatchSnapshot()
})
