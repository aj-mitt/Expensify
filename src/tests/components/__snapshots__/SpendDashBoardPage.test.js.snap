// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render spend dashboard 1`] = `
<div>
  <Connect(SpendingSummary) />
  <Connect(SpendingListFilters) />
  <Connect(SpendingList) />
</div>
`;
